### ISSUE TITLE
Update HTML title and description

### ISSUE THREAD

**Issue by Sideboard** [fortran-lang/playground#45]
Especially for previews the meta info is used. It should better reflect the Fortran context:

```
<title>Playground</title>
<meta name="description" content="Web site created using create-react-app"/>
```

### Suggestions

Title: Fortran Playground
Description: Web-based Fortran editor.

Especially the description can probably be a bit longer than I put it here.

![image](https://user-images.githubusercontent.com/1591390/189864677-d9c27f77-7011-49f5-8471-ca5ac3bf5c30.png)

**Comment by Shaikh-Ubaid** [comment]
Was this issue fixed by #53? If it is safe, shall we close this issue?

**Comment by milancurcic** [comment]
Yes, good catch @Shaikh-Ubaid.
